TodoList Working

G4GANCHAUDHARY opened the pull request.

abhishek-dash commented on assignments/todo_list/todo.html:
```diff
@@ -5,9 +5,9 @@
     <body>
         <div>
             <input type='text' name='todo_input'>
-            <button type='submit'>Add Todo</button>
```
You are using two add to do button.why?

G4GANCHAUDHARY replied:
They are not two todo buttons. Its showing one button deleted(-) one button added(+).

abhishek-dash reviewed (changes requested):
Testcases are not passing.Please do the necessary changes..

abhishek-dash commented:
![Screenshot (337)](https://user-images.githubusercontent.com/55476385/144856285-1ee48fff-680e-4141-8c1e-ea7911654b9b.png)

G4GANCHAUDHARY commented:
> Testcases are not passing.Please do the necessary changes..

TBH not checked the testcases earlier cause todo list was working fine...testcases were passing when I removed onclick attribute from button and added a eventlistner to button in script...bizarre test suites!!
